lumin/app.package

```diff
@@ -1,4 +1,4 @@
 DATAS = "digest.sha512.signed" : "." \
-        "bin/" : "bin/" \
-        "resources/" : "resources/"
+        "bin/" : "bin/"\
+        "../resources/" : "resources/"
```

Comment:
don't we also want to include the `assets` folder?

Reply:
Done